Lib/test/pickletester.py

```diff
@@ -16,6 +16,16 @@
 from textwrap import dedent
 from http.cookies import SimpleCookie
 
+try:
+    import _testbuffer
+except ImportError:
+    _testbuffer = None
+
+try:
+    import numpy as np
+except ImportError:
```

Comment:
It might be interested to log a message when the test suite in run in verbose mode to advice to install numpy?

Reply:
I don't know. `test_buffer.py` doesn't do it.